File: ANDROID APPLICATIONS/Xamarin/2014/KANDOU v2.5/KANDOU v1/DataTypes/SubmissionOfKanji.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Android.App;
using Android.Content;
using Android.OS;
using Android.Runtime;
using Android.Views;
using Android.Widget;

namespace KANDOU_v1
{
    [Serializable]
    class SubmissionOfKanji
    {
        public string signs;
        public string meaning;
        public string reading;
        public int priority;

        public SubmissionOfKanji() { }

        public SubmissionOfKanji(string signs, string meaning, string reading, int priority)
        {
            this.signs = signs;
            this.meaning = meaning;
            this.priority = priority;
            this.reading = reading;
        }
    }
}

File: ANDROID APPLICATIONS/2015/Kandou mobile v3 (beta)/KANDOUmobile_V3/src/aplication/data/SubmissionOfKanji.java
package aplication.data;

import java.io.Serializable;

@SuppressWarnings("serial")
public class SubmissionOfKanji implements Serializable{
	public String signs;
    public String meaning;
    public String reading;
    public int priority;

    public SubmissionOfKanji() { }

    public SubmissionOfKanji(String signs, String meaning, String reading, int priority)
    {
        this.signs = signs;
        this.meaning = meaning;
        this.priority = priority;
        this.reading = reading;
    }
}
